src/components/WelcomeScreen/WelcomeScreen.css

```diff
@@ -2,7 +2,7 @@
 #welcome-screen {
   grid-column: 1 / 3;
   grid-template-rows: auto;
-  height: calc(100vh - 60px);
+  height: calc(100vh - 90px);
```

Comment:
This seems to be not addressing the problem. vh is a relative unit, where px is absolute. May be change the welcome screen to be 90vh?

Reply:
Header and Footer were given an absolute height to make sure they remain visible all screen sizes. 
https://github.com/s-maheshbabu/silent-alexa/blob/aab40b79e71c52362e752ee8269efa4b623d1004/src/components/App/App.css#L5 

For that reason, we need to subtract an absolute height to calculate the height of the welcome screen.